app/src/pages/inside/membersPage/modals/permissionMapModal/permissionMap/permissionMap.jsx

```diff
@@ -23,110 +17,102 @@ export class PermissionMap extends Component {
     intl: {},
   };
   generateTableCells = (permission) => {
-    const roles = [ADMINISTRATOR, PROJECT_MANAGER, MEMBER, OPERATOR, CUSTOMER];
-    const hasPermission = (
-      <td className={cx('col', 'has-permission')}>
-        <i className={cx('rp-icons', 'rp-icons-check')} />
-      </td>
+    const { intl } = this.props;
+    const roles = [PROJECT_MANAGER, MEMBER, OPERATOR, CUSTOMER];
+    const ownerPermission = (
+      <div title={intl.formatMessage(ROLE_NAMES.ownTitle)} className={cx('permission-container')}>
+        <div className={cx('owner-permission')} />
+      </div>
     );
-    const noPermission = <td className={cx('col')} />;
+    const notOwnerPermission = (
+      <div
+        title={intl.formatMessage(ROLE_NAMES.notOwnTitle)}
+        className={cx('permission-container')}
+      >
+        <div className={cx('not-owner-permission')} />
```

Comment:
You don't need to create one more empty DIV. Just use ::before or::after

Reply:
Fixed